sample/hello.rb

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env ruby
 
 require "curses"
-include Curses
 
 def show_message(message)
-  width = message.length + 6
-  win = Window.new(5, width,
-		   (lines - 5) / 2, (cols - width) / 2)
-  win.box('|', '-')
+  height, width = 5, message.length + 6
+  top, left = (Curses.lines - height) / 2, (Curses.cols - width) / 2
```

Comment:
How about one per line?

That will be more familiar and understandable for new rubyists